scripts/get-dep-lib.sh

```diff
@@ -103,7 +103,7 @@ fi
 
 # FIX FOR ANDROID,
 if [ "$PLATFORM" == "singleAndroid" ] ; then
-	CURRENT_OPENSSL="OpenSSL_1_0_2o"
+	CURRENT_OPENSSL="OpenSSL_1_1_1c"
 # If changing the openSSL version here, make sure to change it in versions.sh also.
```

Comment:
Of topic but, I'm guessing that this was ment to say packaging/android/variables.sh instead of versions.sh. 
Can't we fix this to?

Reply:
Indeed. ACK. Fixed.